packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -66,8 +55,8 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
               }
             }
           },
-          ':hover .ms-SearchBox-iconContainer': {
-            color: semanticColors.inputIconHovered
+          [`:hover .${classNames.iconContainer}`]: {
```

Comment:
looks like this was something coming from the master and you later removed it. You should consider using this pattern instead of the hardcoded string.

Reply:
@Vitalius1  I'll pop that back in and re-push, thanks!